website/src/content/docs/pt-br/formatter/index.mdx

```diff
@@ -89,21 +89,21 @@ const expr =
   ];
 ```
 
-## Differences with Prettier
+## Diferenças com o Prettier
```

Comment:
We move this section "Differences with Prettier" to its own page.
Let's remove it from this file

Reply:
A heads up would have been appreciated. So you're telling me I translated it for nothing?

Reply:
A heads up would have been appreciated. So you're telling me I translated it for nothing?

Reply:
Apologies for that.